src/site/content/en/blog/browser-level-lazy-loading-for-cmss/index.md

```diff
@@ -0,0 +1,283 @@
+---
+layout: post
+title: Browser-level lazy-loading for CMSs
+subhead: Learnings for adopting the standardized loading attribute
+authors:
+  - felixarntz
+date: 2020-11-16
+#updated: 2020-11-16
+hero: hero.jpg
```

Comment:
Please resize to 3200 px width and optimize with tinyjpg.com

Reply:
Done!